sdcm/cluster.py

```diff
@@ -1209,6 +1216,8 @@ def node_setup(node):
         self.log.debug('Setup duration -> %s s', int(time_elapsed))
 
     def run_stress_thread(self, stress_cmd, timeout, output_dir, stress_num=1):
+
+        stress_cmd = stress_cmd + " -log file=/tmp/cs.log"
```

Comment:
I believe we don't need to output c-s log to an intermediate file. We can pipe the output of the process directly to the exporter (I'd reckon it's a good idea that the exporter pipes output from c-s to the stdout as well).